packages/cli/src/commands/pull.ts

```diff
@@ -30,7 +37,8 @@ const help = () => {
     'DIR'
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
-    --env [filename]               The file to write Development Environment Variables to [.env]
+    --env-file [filename]          The file to write Development Environment Variables to [.env]
```

Comment:
The line for `--env-file` need to be removed. It no longer applies. I'll open a new PR for that.

Reply:
https://github.com/vercel/vercel/pull/7647